src/testRunner/unittests/helpers.ts

```diff
@@ -118,23 +118,23 @@ export function createTestCompilerHost(texts: readonly NamedSourceText[], target
     });
     if (useCaseSensitiveFileNames === undefined) useCaseSensitiveFileNames = ts.sys && ts.sys.useCaseSensitiveFileNames;
     const getCanonicalFileName = ts.createGetCanonicalFileName(useCaseSensitiveFileNames);
-    const filesByPath = ts.mapEntries(files, (fileName, file) => [ts.toPath(fileName, "", getCanonicalFileName), file]);
+    const filesByPath = ts.mapEntries(files, (fileName, file) => [ts.toPath(fileName, "/", getCanonicalFileName), file]);

```

Comment:
This is just making sure the current directory and file paths are operating correctly and we are testing right things. (Kind of unrelated but needed to ensure things are working correctly for this PR)